scripts/source/dispatch_product.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
+
+import argparse
+parser = argparse.ArgumentParser()
+parser.add_argument("repo",
+                    help="Product repository",
+                    type=str)
+parser.add_argument("workspace",
+                    help="GitHub workplace path",
+                    type=str)
+
+args = parser.parse_args()
+
+import os
+
+commands = {'EnterpriseDB/cloud-native-postgresql':
+            f'{args.workspace}/destination/scripts/source/process-cnp-docs.sh {args.workspace}/source {args.workspace }/destination',
+            'EnterpriseDB/fe':
+            f'mkdir -p {args.workspace}/destination/icons-pkg && \
+              cp -fr utils/icons-placeholder/output/* {args.workspace}/destination/icons-pkg/',
+            'EnterpriseDB/LiveCompare':
+            f'node {args.workspace}/destination/scripts/source/livecompare.js'
+            }
+            
+
+cmd = commands[args.repo]
```

Comment:
Error handling could be a bit better here: if the source repo hasn't been added to this script yet, this'll fail with a `KeyError`. Should probably check to see if the command exists in the dictionary and fail with something descriptive (as in the old switch fallback) when it does not.

Reply:
Oh yeah. I thought of that earlier, but it slipped my mind. It might not hurt do do something with the return code at the end of the script too in case the command fails.